File: python/power.py
#power of a number
def power(n, k) :
    if k == 0 :
        return 1
    else :
        return n * power(n, k-1)


n = input("Enter the base : ")
k = input("Enter the power : ")
#print("n^k= flag")
print(str(n) + "^" + str(k))
print(power(n, k))

File: c++_codes/power.cpp
#include<iostream>
using namespace std;
int power(int, int);
int main(){
	int a, b;
	cout<<"Enter the base : ";
	cin>>a;
	cout<<"Enter the power : ";
	cin>>b;
	cout<<power(a, b)<<"\n";
}
int power(int p, int q){
	if(q==1)
		return p;
	else 
		return p*power(p, q-1);
}
